### PR TITLE
Try/test answer from issue in tox repository.

### PR DESCRIPTION
**Pull request by priti-ashvin-shah-ibm**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -27,6 +27,7 @@ commands =
 
 [testenv:docs]
 basepython = python3.9
+skip_install = true
 envdir = .tox/docs
 passenv = DOCS_FROM_MASTER QISKIT_DOCS_BUILD_TUTORIALS
```
Comment (jugmac00) [inline tox.ini:32]:
```suggestion
passenv = DOCS_FROM_MASTER, QISKIT_DOCS_BUILD_TUTORIALS
```

Also see https://tox.wiki/en/latest/faq.html#tox-4-changed-ini-rules

File: tox.ini [hunk 2]
```diff
@@ -32,8 +32,8 @@ envdir = .tox/docs
 passenv = DOCS_FROM_MASTER, QISKIT_DOCS_BUILD_TUTORIALS
 deps =
   -r requirements-dev.txt
-allowlist_externals touch
 commands =
+  allowlist_externals touch
   touch {toxinidir}/docs/.buildingdocs
   sphinx-build -b html {posargs} {toxinidir}/docs/ {toxinidir}/docs/_build/html
```
Comment (jugmac00) [inline tox.ini:38]:
```suggestion
allowlist_externals = touch
commands =
  touch {toxinidir}/docs/.buildingdocs
  sphinx-build -b html {posargs} {toxinidir}/docs/ {toxinidir}/docs/_build/html
```

Comment (priti-ashvin-shah-ibm) [reply tox.ini:38]: https://github.com/Qiskit/qiskit-metal/pull/893/commits/bfe28e7fd4073283e184b80d9be52e1bf85611bc

I did try the syntax that you proposed.  That had also failed.

Comment (priti-ashvin-shah-ibm) [reply tox.ini:38]: I do appreciate your help.  Thank you!!

Comment (jugmac00) [reply tox.ini:38]: You forgot the `=` / the equals sign.

I suggested the correct fix - as a code snippet. On GitHub you could even click on a button and accept it - you do not need to update the code yourself. If you want to update the code yourself, please make sure to copy/paste it correctly. And :crossed_fingers: that it works soon. It is one of these days :-)

File: tox.ini [hunk 3]
```diff
@@ -27,16 +27,19 @@ commands =
 
 [testenv:docs]
 basepython = python3.9
+skip_install = true
```
Comment (jugmac00) [inline tox.ini:30]: Your project has a little bit of a strange installation process. In your `setup.py` you define `requirements.txt` as `install_requires`. Both files server a different purpose, and usually should not be used like that.

With the above line you skip the installation. And as your requirements including `PySide2` are listed in the `requirements.txt`, it won't be installed.

When building the documentation, you import your application, see
https://github.com/Qiskit/qiskit-metal/blob/2f6a5f779e18f8940536743e479d39db08c22eab/docs/conf.py#L33

And your application tries to import `PySide2` - so it goes boom.

For me it looks like you need to delete the above line.

FWIW - I cannot build the documentation locally, even with the above line deleted.

If I may give you a tip: Python packaging has a steep learning curve, but now there is a really good book out there - which I wished would have existed 15 years ago:
https://www.manning.com/books/publishing-python-packages

**Comment by priti-ashvin-shah-ibm** [comment]
Close this pull request for pull request #898 .

**Pull request closed without merging**